src/Network/Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.cs

```diff
@@ -80,5 +80,14 @@ public void TestVirtualNetworkGatewayVpnDeviceConfigurationScripts()
         {
             TestRunner.RunTestScript("Test-VirtualNetworkGatewayConnectionVpnDeviceConfigurations");
         }
+
+        [Fact]
+        [Trait(Category.RunType, Category.LiveOnly)]
```

Comment:
test uses Storage cmdlets that don't work in playback. I have added recorded session Json file for this test. But I will have to LiveOnly Category.

Reply:
Another option would be to make the storage command execution dependent on the test mode, provided you aren't using storage cmdlet output in the rest of the test.

Reply:
Added check in tests scripts for storage command execution dependent on the test mode.